team/kehan-yang.yaml

```diff
@@ -0,0 +1,20 @@
+- title: Kehan Yang
+  avatar: https://avatars.githubusercontent.com/u/32207757?v=4
+  role: Computational Geoscientist
+  organizations:
+    - name: M3 Works LLC
+  bio: Kehan is a computational geoscientist at M3 Works LLC, specilizing in operational snow modelings. 
```

Comment:
Should have noticed that earlier, but multi-line Strings in Yaml need to be started with a `>`
[Here an example from another team member](https://github.com/uwhackweek/event-page-2024/pull/34/files#diff-3d20d2961fbdb5fc302e636e66b85406bba688ed99dc7ec6b6656bded69b9274R6)

Reply:
Thanks!